core-services/template-deprecation/_allowlist.yaml

```diff
@@ -706,6 +706,9 @@ templates:
     unknown_blocker:
       description: unknown
       jobs:
+        promote-release-openshift-machine-os-content-e2e-aws-4.5:
```

Comment:
Why is this needed to modify an existing job?

Reply:
Because previously the job did not mount the template at all although it attempted to use it (see the added `prow-job-cluster-launch-installer-e2e` CM mount). `template-deprecator` uses template CM to identify template-using jobs so this looks legit to me